server/pages/index.js

```diff
@@ -492,6 +492,21 @@ function render404( request, response ) {
 	response.status( 404 ).send( renderJsx( '404', ctx ) );
 }
 
+function renderServerError( err, req, res, next ) {
+	if ( err ) {
```

Comment:
I know this was simply moved over from [`server/render`](https://github.com/Automattic/wp-calypso/pull/26655/files#diff-d92d930a21711a960b300aeb231d9056L193), but the check doesn't seem to make a lot of sense -- it essentially verifies there's at least one arg, but we're referencing all the other args regardless (most notably, `next()` would fail quite badly if it wasn't present). I'd opt for removing it.